sdk/js/package.json

```diff
@@ -11,15 +11,15 @@
   "sideEffects": false,
   "repository": "https://github.com/certusone/wormhole/tree/main/sdk/js",
   "scripts": {
-    "build-contracts": "npm run build --prefix ../../ethereum && node scripts/copyContracts.js && typechain --target=ethers-v5 --out-dir=src/ethers-contracts contracts/*.json",
```

Comment:
Is the typechain command to generate the contract interface ran from somewhere else now?

Reply:
right here! https://github.com/wormhole-foundation/wormhole/blob/a630f52eed319ee71b6950537ad6eb187e2f5c1e/ethereum/package.json#L21

(in the PR that this one is based off of)
https://github.com/wormhole-foundation/wormhole/pull/3038